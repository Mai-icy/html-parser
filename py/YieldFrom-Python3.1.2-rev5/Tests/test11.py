#
#   Test 'value' attribute of StopIteration exception
#

def pex(e):
	print("%s: %s" % (e.__class__.__name__, e))
	print("value =", e.value)

e = StopIteration()
pex(e)
e = StopIteration("spam")
pex(e)
e.value = "eggs"
pex(e)
