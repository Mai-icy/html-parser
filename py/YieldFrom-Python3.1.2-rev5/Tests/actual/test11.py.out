StopIteration: 
value = None
StopIteration: spam
value = spam
StopIteration: spam
value = eggs
